wps-common/src/main/java/au/org/aodn/aws/geoserver/client/SubsetParameters.java

```diff
@@ -1,4 +1,4 @@
-package au.org.emii.geoserver.client;
+package au.org.aodn.aws.geoserver.client;
```

Comment:
We use emii everywhere else so just using aodn here is inconsistent and not sure why aws is relevant here.

Reply:
Just makin' stuff up...  (will fix)

Reply:
(won't fix) - backlog worthy